.github/workflows/update-generated-code.yaml

```diff
@@ -0,0 +1,77 @@
+name: Update Generated Docs
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The new tolgee platform version to generate docs from."
```

Comment:
Couldn't we run this periodically, so we don't forget to run this? (e.g. like oss friends)